packages/src/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
```

Comment:
I guess this comment block should be removed.

Reply:
I left this comment but added this one under it `// Import blocks here`